packages/crowdstrike/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.56.1"
+  changes:
+    - description: Expose missing number_of_workers settings to the user.
```

```suggestion
    - description: Expose missing "Number of Workers" settings to the user.
```
Users should not be expected to read code to understand change logs.